app/views/components/_share-links.html.erb

```diff
@@ -0,0 +1,18 @@
+<% title ||= 'Share this page' %>
+
+<div class="app-c-share-links-wrapper">
+  <h2><%= title %></h2>
+  <div class="app-c-share-links" data-module="track-share-button-clicks">
+    <%= link_to share_urls[:facebook],
```

Comment:
I think this:
https://github.com/alphagov/government-frontend/blob/aaib-dates/app/presenters/shareable.rb#L4-L9

Could become:
```ruby
def share_links
    {
      title: 'Share this page',
      facebook_href: facebook_share_url,
      twitter_href: twitter_share_url
    }
  end
```

Like this:
https://github.com/alphagov/government-frontend/blob/master/app/presenters/metadata.rb#L5